pkg/resources/account_password_policy_attachment.go

```diff
@@ -0,0 +1,85 @@
+package resources
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/helpers"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+var accountPasswordPolicyAttachmentSchema = map[string]*schema.Schema{
+	"password_policy": {
+		Type:        schema.TypeString,
+		Required:    true,
+		ForceNew:    true,
+		Description: "Qualified name (`\"db\".\"schema\".\"policy_name\"`) of the password policy to apply to the current account.",
```

Comment:
if we are going to do this, we should probably make data source(s) to help make it easier for people to make fully qualified names from db / schema / object names

Reply:
I added the qualified name as an attribute of the password policy resource, but the data source is a good idea too.